infrastructure/functions/outputs.tf

```diff
@@ -29,6 +30,12 @@ output "lambda_post_confirmation_trigger"{
   }
 }
 
+output "lambda_pre_signup_trigger"{
+  value = {
+     "verify_users_with_same_email_lambda_function": aws_lambda_function.verify_users_with_same_email_lambda_function.invoke_arn
```

Comment:
You need arn, instead of invoke_arn
Please move this output in one output lambda_trigger with post register and pre register values